models/articles.models.js

```diff
@@ -20,8 +20,11 @@ exports.selectArticle = (articleId) => {
   return db
     .query(
       `
-      SELECT * FROM articles 
-      WHERE article_id = $1`,
+      SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, articles.body, COUNT(comments.article_id) AS comment_count
+      FROM articles
+      FULL OUTER JOIN comments ON articles.article_id = comments.article_id
+      WHERE articles.article_id = $1
+      GROUP BY articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, articles.body`,
```

Comment:
<img width="255" alt="image" src="https://github.com/nicomaz/news-api/assets/99129306/6739224c-d4a8-487b-be75-6e1d9d8862d6">
This query definitely does the job, but I'd say that the use of a FULL OUTER JOIN is slightly off, because that will include all articles and all comments - even those that do not match. I'd argue a LEFT JOIN (the default one) works better here